BendAssist.App/BendAssists/CornerClose.cs

```diff
@@ -21,77 +21,81 @@ public sealed class CornerClose : BendAssist {
-               if (mStepLines.IndexOf (pLine) % 2 == 0) {
-                  PLine translatedLine = pLine.Orientation switch {
-                     Horizontal => pLine.EndPoint.Y < mPart.Centroid.Y ? (PLine)pLine.Translated (0, lineShift) : (PLine)pLine.Translated (0, -lineShift),
-                     Vertical => pLine.EndPoint.X < mPart.Centroid.X ? (PLine)pLine.Translated (lineShift, 0) : (PLine)pLine.Translated (-lineShift, 0),
-                     _ => null!
-                  };
-                  mNewLines.Add (new PLine (translatedLine.StartPoint, translatedLine.EndPoint, ++mIndex));
-               } else {
-                  (PLine extrudedLine, PLine trimmedLine) = pLine.Orientation switch {
-                     Horizontal => pLine.StartPoint.X < mPart.Centroid.X ? ((PLine)pLine.Trimmed (-halfBA, 0, 0, 0).Translated (0, halfBD), (PLine)pLine.Trimmed (lineShift, 0, pLine.Length - halfBA, 0))
-                                                                        : ((PLine)pLine.Trimmed (halfBA, 0, 0, 0).Translated (0, -halfBD), (PLine)pLine.Trimmed (-lineShift, 0, -(pLine.Length - halfBA), 0)),
-                     Vertical => pLine.StartPoint.Y < mPart.Centroid.Y ? ((PLine)pLine.Trimmed (0, -halfBA, 0, 0).Translated (-halfBD, 0), (PLine)pLine.Trimmed (0, lineShift, 0, pLine.Length - halfBA))
-                                                                      : ((PLine)pLine.Trimmed (0, halfBA, 0, 0).Translated (halfBD, 0), (PLine)pLine.Trimmed (0, -lineShift, 0, -(pLine.Length - halfBA))),
-                     _ => (null!, null!)
-                  };
-                  mNewLines.Add (new PLine (trimmedLine.StartPoint, trimmedLine.EndPoint, ++mIndex));
-                  mNewLines.Add (new PLine (trimmedLine.EndPoint, extrudedLine.StartPoint, ++mIndex));
-                  mNewLines.Add (new PLine (extrudedLine.StartPoint, extrudedLine.EndPoint, ++mIndex));
-               }
+               (PLine extrudedLine, PLine trimmedLine) = pLine.Orientation switch {
+                  Horizontal => pLine.StartPoint.X < mPart.Centroid.X ? ((PLine)pLine.Trimmed (-halfBA, 0, 0, 0).Translated (0, halfBD), (PLine)pLine.Trimmed (lineShift, 0, pLine.Length - halfBA, 0))
+                                                                     : ((PLine)pLine.Trimmed (halfBA, 0, 0, 0).Translated (0, -halfBD), (PLine)pLine.Trimmed (-lineShift, 0, -(pLine.Length - halfBA), 0)),
+                  Vertical => pLine.StartPoint.Y < mPart.Centroid.Y ? ((PLine)pLine.Trimmed (0, -halfBA, 0, 0).Translated (-halfBD, 0), (PLine)pLine.Trimmed (0, lineShift, 0, pLine.Length - halfBA))
+                                                                   : ((PLine)pLine.Trimmed (0, halfBA, 0, 0).Translated (halfBD, 0), (PLine)pLine.Trimmed (0, -lineShift, 0, -(pLine.Length - halfBA))),
+                  _ => (null!, null!)
+               };
+               mNewLines.Add (new PLine (trimmedLine.StartPoint, trimmedLine.EndPoint, ++mIndex));

```

Comment:
use new () instead of new PLine.
store start and end point of trimmedPline in a variable and use them through out this method